encode_test.go

```diff
@@ -460,7 +461,7 @@ func TestByteaOutputFormats(t *testing.T) {
 		return
 	}
 
-	testByteaOutputFormat := func(f string) {
+	testByteaOutputFormat := func(f string, s bool) {
```

Comment:
I don't like the name "s" here; it's not at all clear what it means.  Maybe "prepare" or "prepared" or "usePrepared" would be more clear?